runtime/vm/jvminit.c

```diff
@@ -882,14 +882,14 @@ freeJavaVM(J9JavaVM * vm)
 		vm->tlsPool = NULL;
 	}
 
-	if (NULL != vm->globalContinuationCacheArray) {
-		for (U_32 i = 0; i < vm->continuationArraySize; i++) {
-			if (NULL != vm->globalContinuationCacheArray[i]) {
-				freeJavaStack(vm, vm->globalContinuationCacheArray[i]->stackObject);
-				j9mem_free_memory(vm->globalContinuationCacheArray[i]);
+	if (NULL != vm->continuationT2Cache) {
+		for (U_32 i = 0; i < vm->continuationT2Size; i++) {
+			if (NULL != vm->continuationT2Cache[i]) {
+				freeJavaStack(vm, vm->continuationT2Cache[i]->stackObject);
+				j9mem_free_memory(vm->continuationT2Cache[i]);
```

Comment:
Is there any other structure such as `decompilationStack` requiring explicit freeing?